File: python/loop/1064.py
#1064 POSITIVOS E MÉDIA
positivos = 0
soma = 0

for _ in range(6):
    i = float(input())
    if i > 0:
        positivos += 1
        soma += i
        media = soma/positivos

print(f"{positivos} valores positivos")
print(f"{media:.1f}")

File: javascript/1064.js
/*Positivos e Média*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let lines = input.split('\n');

let valoresPositivos = 0;
let somaNum = 0;
  
for(let i = 1; i <= 6; i++){
  numeros = Number(lines.shift());
      if(numeros > 0.0){
        valoresPositivos++;
        somaNum += numeros;
}
}
media = (somaNum/valoresPositivos);

console.log(valoresPositivos + " valores positivos");
console.log(media.toFixed(1));